docker/proxy/nginx.conf

```diff
@@ -0,0 +1,35 @@
+user nobody nogroup;
+worker_processes auto;
+
+events {
+  worker_connections 512;
+}
+
+http {
+
+  server {
+    listen 8080;
+    server_name "";
+
+    location / {
+      proxy_pass					http://docker2.pcic.uvic.ca:8000;
```

Comment:
Ah yes, this is definitely a good idea. For simplicity, I think nginx's solution will work just fine.